userspace/falco/app/actions/configure_interesting_sets.cpp

```diff
@@ -177,6 +168,27 @@ static void select_event_set(falco::app::state& s, const libsinsp::events::set<p
 		}
 	}
 
+	/* if a custom set is specified (positive, negative, or both), we attempt
```

Comment:
Set repairing should happen at the last stage, so that we make sure users don't unwillingly delete an essential syscall and Falco can recover by repairing it (if configured to do so). The test cases now assert this as well by negating `bind` and then repairing it.